cricPredict/Contest/tasks.py

```diff
@@ -0,0 +1,59 @@
+from celery.schedules import crontab
+from celery.task import task, periodic_task
+import requests
+import json
+from .models import League, Match
+from datetime import datetime
+from django.utils.timezone import get_current_timezone
+
+
+@periodic_task(run_every=(crontab(minute='*')), name="fetch_data", ignore_result=True)
+def fetch_data():
+    try:
+        url = "https://rapidapi.p.rapidapi.com/series.php"
+
+        headers = {
+            'x-rapidapi-key': "94d65a12cbmsh65295dae3718214p1ee3b8jsn335c8c3f26ef",
```

Comment:
if this is the header unique for your user/API usage, this shouldn't be pushed to VCS in a public repo